handbook/engineering.md

```diff
@@ -117,5 +117,19 @@ There are several locations in Fleet's public and internal documentation that ca
 
 2. The [Internal FAQ](https://docs.google.com/document/d/1I6pJ3vz0EE-qE13VmpE2G3gd5zA1m3bb_u8Q2G3Gmp0/edit#heading=h.ltavvjy511qv) document.
 
+## Slack directory
+
+These are the Slack channels the core engineering team maintains. If the channel has a [directly responsible individual](./people.md#directly-resonsible-individuals) (**DRI**), they will be specified. These people are responsible for keeping up with all new messages, even if they aren't mentioned. 
+
+- **#g-core-engineering** - **DRI**: Zach Wasserman
+- **#help-oncall**
+- **#help-golang**
+- **#help-qa**
+- **#help-frontend**
+- **#_pov-environments**
```

Comment:
- DRI of #help-oncall is Zach Wasserman
- DRI of #help-golang is Zach Wasserman
- DRI of #help-qa is Reed Haynes (@xpkoala)
- DRI of #help-frontend is @lukeheath 
- DRI of #_pov-environments is Ben Edwards  (@edwardsb)